Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Triggered_Domain.json

```diff
@@ -0,0 +1,32 @@
+{
+    "id": "incident_triggereddomain",
+    "version": -1,
+    "modified": "2023-11-09T22:08:00.46683194+05:30",
+    "name": "Triggered domain",
+    "ownerOnly": false,
+    "description": "Triggered domain",
+    "cliName": "triggereddomain",
+    "type": "shortText",
```

Comment:
@syed-loginsoft - I thought we discussed the _Triggered domain_ will be a list of domains.

Reply:
@kgal-pan Should @syed-loginsoft change the type to 'list'?

Reply:
@melamedbn - According to what we discussed on our call last week, the field should be a dynamic list.